MK404.cpp

```diff
@@ -324,6 +326,7 @@ int main(int argc, char *argv[])
 		cout << "GL_RENDERER  : " << glGetString(GL_RENDERER) << endl;
 		cout << "GLEW_VERSION : " << glewGetString(GLEW_VERSION) << endl;
 		//cout << "GLSL VERSION : " << glGetString(GL_SHADING_LANGUAGE_VERSION) << endl;
+#if !defined(__APPLE__)
```

Comment:
This will segfault if GL  is not initialized correctly. On Linux, this is done by glewInit(), we may need an extra call for OSX.

Reply:
My OpenGL experience is exactly zero

Reply:
Fa enough, my OSX experience matches your OpenGL 😄